swarm-derive/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-## 0.33.0 
+## 0.34.0
+
+- Add `from_swarm::NewExternalAddrOfPeer` event.
```

Comment:
I think this should say something along the lines of: "Generate code for ..."

Reply:
:heavy_check_mark: